web-ts/src/RootSwitcher.tsx

```diff
@@ -63,7 +63,7 @@ const RootSwitcher = ({ setContentLanguage }: PropsType): ReactElement => {
 
   return (
     <Switch>
-      <Route exact path={RoutePatterns[LANDING_ROUTE]} component={LandingPage} />
+      <Route exact path={RoutePatterns[LANDING_ROUTE]} render={props => <LandingPage cities={cities} {...props} />} />
```

Comment:
I think this could cause performance issues as we are passing here a closure instead of a static component. We once had a very restrictive eslint rule for this. Maybe its worth here as we are quite low in the component tree.

Reply:
Actually not sure why we don't just use normal react children for rendering as recommended since both `component` and `render` are apparently more or less legacy since hooks came out:
https://reactrouter.com/web/api/Route/route-render-methods

However, it seems like `render` has no performance problems even though supplying a function:
`This allows for convenient inline rendering and wrapping without the undesired remounting explained above.`

But I guess we should use 
```
<Route ...>
  <LandingPage {...props} />
</Route>
```
anyway...

Reply:
Seems like we loose the types which params are available/passed. Will leave this for now as is.